### PR TITLE
help util: replace jtextutils with asciitable

### PR DESCRIPTION
**Pull request by mpollmeier**
* main motivation: jtextutils became vulnerable because the domain
`com.massisframework` is available for sale...
* also: asciitable is better IMO

### CODE REVIEW

**Comment by mpollmeier** [comment]
cc @Ferada

File: traversal/src/main/scala/overflowdb/traversal/help/Table.scala [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
+      table.addRule()
+      table.addRow(columnNames.asJava)
+      table.addRule()
+      if (rows.nonEmpty) {
+        rows.map(_.asJava).foreach(table.addRow)
+      }
+      table.addRule()
+      table.getContext.setGridTheme(TA_GridThemes.FULL)
+      table.setTextAlignment(TextAlignment.LEFT)
+      table.render(width)
```
Comment (maltek) [inline traversal/src/main/scala/overflowdb/traversal/help/Table.scala:25]: what happens when the contents are wider than 120 characters? (e.g. dataflows are wider than that all the time)

Comment (mpollmeier) [reply traversal/src/main/scala/overflowdb/traversal/help/Table.scala:25]: It get's split up into several lines. There's quite a few options, I went for left alignment for the text. 
https://github.com/vdmeer/asciitable/?tab=readme-ov-file#text-alignment-1

Comment (mpollmeier) [reply traversal/src/main/scala/overflowdb/traversal/help/Table.scala:25]: (previously, the table would just expand in width to the widest cell)

**Comment by mpollmeier** [comment]
For comparison: in joern:
old: 
image
image


new:
image
image

**Comment by mpollmeier** [comment]
n.b. I'm working on a followup change that'll pass down the terminal size to the text table

**Review by maltek — APPROVED** [review]
ok, that does look nicer :+1:

**Comment by Ferada** [comment]
Thanks for looking into that :+1:

**Pull request merged**